loader/extends.go

```diff
@@ -60,6 +60,8 @@ func applyServiceExtends(ctx context.Context, name string, services map[string]a
 	if !ok {
 		return s, nil
 	}
+	opts.ProcessEvent("extends", name)
```

Comment:
in addition to service name, would be useful to track the file so we can detect all files involved in a compose model
`Listener`'s metadata parameter could be declared a `map[string]any` so it is easy to pass more than just one attribute here, and add more as this evolves without impacting consumers

Reply:
this was a leftover from a test :) I intended to pass the `extends` (the whole extends metadata from the service) I will change this. I left a `any` in case for some reason in the future someone does not want to necessarily pass a `map[string]any`. What do you think @ndeloof ?

Reply:
the benefit of a `map[string]any` is that this allows to "add more" without impact on previously reported keys. Using `any` the consumer would need to adjust to any change that's made to the original struct